### PR TITLE
Moving wrapper script args

### PR DESCRIPTION
**Pull request by ChristinaLK**
Making the wrapper script args section required, and rewriting to use with the blast example instead of OpenBUGS

### CODE REVIEW

**Comment by ChristinaLK** [comment]
completely untested. 😂

File: docs/materials/software/part1-ex3-arguments.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+status: testing
+---
+
+<style type="text/css"> pre em { font-style: normal; background-color: yellow; } pre strong { font-style: normal; font-weight: bold; color: #008; } </style>
+
+Software Exercise 4.1: Passing Arguments Through the Wrapper Script
```
Comment (lmichael107) [inline docs/materials/software/part1-ex3-arguments.md]:
```suggestion
Software Exercise 1.3: Passing Arguments Through the Wrapper Script
```

**Review by lmichael107 — CHANGES_REQUESTED** [review]

File: docs/materials/index.md [hunk 2]
```diff
@@ -53,7 +54,6 @@ Slides will be posted here
 
 ### Software Exercises 4: Bonus Exercises
 
-- [Exercise 4.1: Using Arguments With Wrapper Scripts](software/part4-ex1-arguments.md)
 - [Exercise 4.2: Additional Python ](software/part4-ex2-python-extras.md)
```
Comment (lmichael107) [inline docs/materials/index.md]:
```suggestion
- [Exercise 4.1: Additional Python ](software/part4-ex2-python-extras.md)
```

File: docs/materials/index.md [hunk 3]
```diff
@@ -53,7 +54,6 @@ Slides will be posted here
...
 - [Exercise 4.3: Using Software in a Docker Container](software/part4-ex3-docker.md)
```
Comment (lmichael107) [inline docs/materials/index.md]:
```suggestion
- [Exercise 4.2: Using Software in a Docker Container](software/part4-ex3-docker.md)
```

File: docs/materials/index.md [hunk 4]
```diff
@@ -53,7 +54,6 @@ Slides will be posted here
...
 - [Exercise 4.4: Building Your Own Docker Container (Beta)](software/part4-ex4-docker-build.md)
```
Comment (lmichael107) [inline docs/materials/index.md]:
```suggestion
- [Exercise 4.3: Building Your Own Docker Container (Beta)](software/part4-ex4-docker-build.md)
```

**Review by lmichael107 — COMMENTED** [review]
Exercises 4.2-4.4 corrected to 4.1-4.3, but their internal titles still need to be updated. Will merge (so this can all be tested) and then make the changes in a new pull request.

**Review by lmichael107 — APPROVED** [review]

**Pull request merged**